src/nexus/NexusBridgeDAO.sol

```diff
@@ -0,0 +1,49 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+import {NexusBaseBridge} from "./NexusBaseBridge.sol";
+
+/**
+ * @title Nexus Bridge DAO Contract
+ * @author RohitAudit
+ * @dev This contract enables DAO governing body of the rollup to claim the staking rewards
+ * from the bridge contract and use it for Ecosystem incentives etc.
+ */
+abstract contract NexusBridgeDAO is NexusBaseBridge {
+    uint256 public rewardsClaimed;
+    // to be changed rollup DAO
+    address public constant DAO = 0x14630e0428B9BbA12896402257fa09035f9F7447;
```

Comment:
@RohitAudit 
is this a multi sig EOA address or Contract address?

Reply:
This can be any address. For now it is EOA address